buildkite/builds.go

```diff
@@ -27,7 +28,7 @@ type CreateBuild struct {
 
 	// Optional fields
 	Author                      Author            `json:"author,omitempty" yaml:"author,omitempty"`
-	CleanCheckout 				bool 			  `json:"clean_checkout,omitempty" yaml:"clean_checkout,omitempty"`
+	CleanCheckout               bool              `json:"clean_checkout,omitempty" yaml:"clean_checkout,omitempty"`
```

Comment:
gofmt did this, let me know if you'd like me to change it back

Reply:
Should be OK - looks fine with me